frontend/components/StoreSelectorButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { View, TouchableOpacity } from "react-native";
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import Svg, { Path } from 'react-native-svg';
+
+export default function StoreSelector() {
+    return (
+        <View style={{flexDirection: "row", paddingRight: 10}}>
+            <TouchableOpacity style= {{margin: 0, borderColor: "#00E676", borderWidth: 1, borderTopLeftRadius: 20, borderBottomLeftRadius: 20, borderRightWidth: 0}}>
```

Comment:
Having the styles inline like this makes it hard to read. I think it would be better if we created a StyleSheet at the bottom